[2022-09-05 03:07:25,912] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-02T00:00:00+00:00 [queued]>
[2022-09-05 03:07:26,193] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2022-08-02T00:00:00+00:00 [queued]>
[2022-09-05 03:07:26,213] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:26,214] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2022-09-05 03:07:26,222] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 03:07:26,426] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2022-08-02T00:00:00+00:00
[2022-09-05 03:07:26,432] {standard_task_runner.py:52} INFO - Started process 219 to run task
[2022-09-05 03:07:26,495] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2022-08-02T00:00:00+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp7csluxn4', '--error-file', '/tmp/tmpb_m12xnn']
[2022-09-05 03:07:26,640] {standard_task_runner.py:77} INFO - Job 14: Subtask second_task
[2022-09-05 03:07:27,060] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2022-08-02T00:00:00+00:00 [running]> on host 2b8e628fc16c
[2022-09-05 03:07:27,542] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T00:00:00+00:00
[2022-09-05 03:07:27,543] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-09-05 03:07:27,545] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2022-09-05 03:07:27,607] {bash.py:169} INFO - Output:
[2022-09-05 03:07:27,614] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2022-09-05 03:07:27,616] {bash.py:177} INFO - Command exited with return code 0
[2022-09-05 03:07:28,041] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20220802T000000, start_date=20220905T030725, end_date=20220905T030728
[2022-09-05 03:07:28,632] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-05 03:07:28,728] {local_task_job.py:146} INFO - Task exited with return code 0
